### PR TITLE
Release/v0.20.0

### PR DESCRIPTION
**Pull request by codealchemist**
@feross maybe we should also update the authors file.
If I'm right we have 3 new contributions from people not already there:
- Borewit
- terryhau
- vitorgalvao

Also, we might want to add these two unmerged fixes:
#1314 
#1315 

Cheers!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+### Added
+- Added support for additional audio extensions: 'aiff', 'ape', 'mp2', 'oga', 'opus', 'wma' (#1240)
+
+### Changed
+
+- README.md: added cask instructions (#1364)
+- Displaying filename while music metadata is being downloaded (#1361)
+- Improved the poster selection for audio/music based torrent (#1334)
+- Removed 'video-on-top' from VLC player launch options (#1286)
+- Replaced dependency `musicmetadata` with `music-metadata` (#1240)
```
Comment (feross) [inline CHANGELOG.md]: I usually skip mentioning internal changes like this that the end user will not notice

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+
+### Fixed
+
+- Added missing key to torrent status element (#1166)
```
Comment (feross) [inline CHANGELOG.md]: Same with this line :)

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "webtorrent-desktop",
   "description": "WebTorrent, the streaming torrent client. For Mac, Windows, and Linux.",
-  "version": "0.19.0",
+  "version": "0.20.0",
```
Comment (feross) [inline package.json]: The version will be incremented during the release process, so I'll undo this change in this PR!

**Comment by codealchemist** [comment]
Oh, just saw you merged it @feross.
I was about to add a reference for:
- Fixed external calls failing silently to open magnets links on Linux (#1367)

And remove those unwanted references from the changelog.
Also I would like to suggest _Leo Dutra_, who helped fixing that one, as a new entry on the authors file :)
If you like I can still update this and open the PR again.

Thx!

**Comment by feross** [comment]
@codealchemist Ah, no worries. I can fix up the changelog before release.

The AUTHORS file is automatically generated by running `npm run update-authors`, so there's no need to manually update the file. :)

**Comment by codealchemist** [comment]
Oh, I totally forgot we had that!
Cool!
😄

**Comment by feross** [comment]
Aaaaand, 0.20.0 is released!

**Comment by codealchemist** [comment]
image

**Pull request merged**